mobile_net/utils/dataset.py

```diff
@@ -0,0 +1,86 @@
+# !/usr/bin/env python
```

Comment:
If we want to do the whole “divide into multiple modules” thing (instead of just writing a few more scripts that we put into the root folder of the repository), *please use the src/module_name/ layout*.

But if you ask me, we should just write a few scripts in the repo root folder for now (we already have `sample_from_video_file.py`, why not just add the next stage(s) beside that?) and only after that think about whether we want to make this a library (which is probably not the case because it's rather restricted to this exact data set/learning task).